query-service-impl/src/main/java/org/hypertrace/core/query/service/pinot/PinotBasedRequestHandler.java

```diff
@@ -113,7 +118,7 @@ public void handleRequest(
       QueryContext queryContext,
       QueryRequest request,
       QueryResultCollector<Row> collector,
-      RequestAnalyzer requestAnalyzer) {
+      RequestAnalyzer requestAnalyzer) throws InvalidProtocolBufferException {
```

Comment:
This would rarely happen. Very low chance actually since this is in memory proto request. We can do try catch inside

Reply:
Fair point. Fixed.